facilitator/Dockerfile

```diff
@@ -38,10 +38,14 @@ ARG BUILD_INFO=unspecified
 ENV OPENSSL_STATIC=true
 
 # This cargo build command must match the one above, or the build cache will not be reused.
-RUN cargo build --manifest-path ./facilitator/Cargo.toml
-# We build in debug mode so the build runs quickly, then strip the binary for size.
-RUN strip facilitator/target/debug/facilitator
+RUN cargo build --manifest-path ./facilitator/Cargo.toml --release
+
+# Build a minimal container from Alpine containing only the stripped binary and
+# no intermediate build artifacts
+FROM alpine:3.14
+
+Run apk add libgcc && apk update
```

Comment:
```suggestion
RUN apk add libgcc && apk update
```

Reply:
I didn't realize that all-caps for directives was optional in Dockerfiles, and I'm a little disappointed.